src/views/Profile/AuthorsList/AuthorsTableHead.tsx

```diff
@@ -7,94 +7,105 @@ import {
+import {
+  AutorsListOrder,
+  AuthorsListSortBy,
+  ISortAutorsList,
+} from '../../../models/experts/types';
 
 interface IContent {
   label: string;
   isSortable: boolean;
-  //   sortKey?: keyof typeof SortBy;
-  //   initialSortOrder?: keyof typeof Order;
+  sortKey?: string;
+  initialSortOrder?: keyof typeof AutorsListOrder;
   tooltip?: string;
-  icon?: JSX.Element;
 }
 
 const content: IContent[] = [
   {
     label: i18n.t(langTokens.admin.id),
-    isSortable: false,
+    sortKey: AuthorsListSortBy.autorId,
+    isSortable: true,
+    initialSortOrder: AutorsListOrder.desc,
   },
   {
     label: i18n.t(langTokens.admin.fullName),
-    isSortable: false,
+    sortKey: AuthorsListSortBy.fullName,
+    isSortable: true,
+    initialSortOrder: AutorsListOrder.desc,
   },
   {
     label: i18n.t(langTokens.admin.region),
-    isSortable: false,
+    sortKey: AuthorsListSortBy.region,
+    isSortable: true,
+    initialSortOrder: AutorsListOrder.desc,
   },
   {
     label: i18n.t(langTokens.admin.city),
-    isSortable: false,
+    sortKey: AuthorsListSortBy.city,
+    isSortable: true,
+    initialSortOrder: AutorsListOrder.desc,
   },
   {
     label: i18n.t(langTokens.admin.dateOfCreation),
-    isSortable: false,
+    sortKey: AuthorsListSortBy.createdAt,
+    isSortable: true,
+    initialSortOrder: AutorsListOrder.desc,
   },
   {
     label: i18n.t(langTokens.admin.dateOfEdition),
-    isSortable: false,
+    sortKey: AuthorsListSortBy.editedAt,
+    isSortable: true,
+    initialSortOrder: AutorsListOrder.desc,
   },
   {
     label: i18n.t(langTokens.admin.actions),
     isSortable: false,
   },
 ];
 
-const AuthorsTableHead: React.FC = () => {
-  //   const classes = useStyles();
-  //   const [boundedSetSort] = useActions([setSort]);
-  //   const {
-  //     sort: { order, sortBy },
-  //   } = useSelector(selectMeta);
-
-  //   const handleSort = (cell: IContent) => {
-  //     const { sortKey, initialSortOrder } = cell;
-  //     let newOrder: keyof typeof Order = initialSortOrder || Order.asc;
+const AuthorsTableHead: React.FC<ISortAutorsList> = (props) => {
+  const classes = useStyles();
+  const [boundedSetSort] = useActions([setSort]);
+  const { order, sortBy } = props;
 
-  //     if (sortKey === sortBy) {
-  //       newOrder = order === Order.desc ? Order.asc : Order.desc;
-  //     }
+  const handleSort = (cell: IContent) => {
+    const { sortKey, initialSortOrder } = cell;
+    let newOrder: keyof typeof AutorsListOrder =
+      initialSortOrder || AutorsListOrder.asc;
 
-  //     boundedSetSort({
-  //       sortBy: sortKey as keyof typeof SortBy,
-  //       order: newOrder,
-  //     });
-  //   };
+    if (sortKey === sortBy) {
+      newOrder =
+        order === AutorsListOrder.desc
+          ? AutorsListOrder.asc
+          : AutorsListOrder.desc;
+    }
+    boundedSetSort({
+      order: newOrder,
+      sortBy: sortKey as keyof typeof AuthorsListSortBy,
```

Comment:
also "autors" without "h". Please check if there are other instances across files you changed anyway.